apps/admin/frontend/src/screens/tally/manual_tallies_form_screen.tsx

```diff
@@ -551,7 +551,7 @@ function ContestForm({
 
   const contest = getContestById(electionDefinition, contestId);
```

Comment:
My guess is that this is an accidental use of `getContestById`, thinking that it was one of the election utils from libs/types (which don't have any caching). I think we should replace this with a simple `find(election, (contest) => contest.id === contestId)`. To prevent this footgun in the future, maybe we limit the exporting of the tabulation lookup functions somehow or rename them, since they are probably intended to only be used in very specific circumstances but they are named similarly to the regular election utils.